scripts/sync_ros_packages.py

```diff
@@ -94,6 +94,13 @@
 
         updates_generator.add_update_element(conf.UpdateElement(**d))
 
+        # The source architecture doesn't provide a Packages file to diff with.
+        # But it should still be added to the updates_generator above so the
+        # source repository is updated.
+        if arch == 'source':
```

Comment:
This looks OK, but you also need to remove the old check at the top of the loop, I believe.

Reply:
:man_facepalming:

Reply:
Fixed via force push of my (apparently) uncommitted changes.